assets/prometheus-k8s/role-specific-namespaces.yaml

```diff
@@ -16,6 +16,14 @@ items:
     - get
     - list
     - watch
+  - apiGroups:
```

Comment:
Seems like this is a cause of failing e2e tests.

Reply:
Yes indeed but also seems like a super weird addition since we did not bump prometheus operator version 🤔 Will look into it.